### DISCUSSION TITLE
Student ID email outdated

### DISCUSSION THREAD

**Discussion by AAGF470** [community/community#119730] [GitHub Education]
### Hi everyone, ID not recognized issue :(

I can't seem to get my student email recognized by the system despite the domain being in the school system. I believe the issue is that the email for my school district has changed slightly, students now use the @students.kleinisd.net 

But the system still has what I remember was the original email domain of @kleinisd.net

![check](https://github.com/community/community/assets/82011993/5d49bf33-34ae-437c-9b38-4ef96c9a88e6)
Subdomains may also be supported by the ID system, I'm just not sure, but something is preventing me from going past the "We require applicants of (my school)  to use one of these school-issued email addresses to apply:" despite my school email with the "@students.kleinisd.net" already being in my account and verified. I have checked another post similar to mine but there was only an internal solution so it does not help me in any way, 

Any help or guidance is appreciated!

**Comment by samus-aran** [comment]
Hi @AAGF470 

We've updated our [Student Developer Pack Application FAQs & Common Rejection Reasons](https://github.com/orgs/community/discussions/111352) discussion. It's packed with valuable information that might just have what you're looking for. I've included the section that I feel would best assist.

- [My selected school states I must have a specific school-issued email address but it’s incorrect, what should I do?](https://github.com/orgs/community/discussions/111352#selected-school-incorrect-details)

Remember to return to your discussion and reply if you need more assistance or mark it as solved 🎉 This will assist others in the community to finding answers more quickly 👍
